src/components/List.js

```diff
@@ -12,11 +12,34 @@ const List = ({ token, results }) => {
             Add Item
           </NavLink>
         </>
-      ) : null}
+      ) : (
+        <form>
```

Comment:
i don't think you need a form here, either. A `form` to me implies submitting some data somewhere.